hack/test-e2e-local.sh

```diff
@@ -28,7 +28,7 @@ ginkgo_flags=
 # This will add a JUnit view above the build log that shows an overview over successful and failed test cases.
 if [ -n "${CI:-}" -a -n "${ARTIFACTS:-}" ]; then
   mkdir -p "$ARTIFACTS"
-  ginkgo_flags="--output-dir=$ARTIFACTS --junit-report=junit.xml"
+  ginkgo_flags="--failFast --output-dir=$ARTIFACTS --junit-report=junit.xml"
```

Comment:
There is `JOB_TYPE` environment variable set by prow.
Possible values are `periodic`, `presubmit`, `batch` and `postsubmit`.

For periodic jobs it is not really necessary to fail fast. Thus, my suggestion is to test the `failFast` option for two weeks if `JOB_TYPE!=periodic` and see if the situation is better or worse.